syncer/sharding_group.go

```diff
@@ -451,19 +454,17 @@ func (k *ShardingGroupKeeper) AddGroup(targetSchema, targetTable string, sourceI
 }
 
 // Init does initialization staff
-func (k *ShardingGroupKeeper) Init(conn *Conn) error {
+func (k *ShardingGroupKeeper) Init() error {
 	k.clear()
-	if conn != nil {
-		k.db = conn
-	} else {
-		db, err := createConn(k.cfg, k.cfg.To, maxDDLConnectionTimeout)
-		if err != nil {
-			return err
-		}
-		k.db = db
+	sgkDB := k.cfg.To
+	sgkDB.RawDBCfg = config.DefaultRawDBConfig(maxDDLConnectionTimeout)
```

Comment:
Is `maxCheckPointTimeout` or `maxDMLConnectionTimeout` more suitable here? @amyangfei

Reply:
IMO `maxCheckPointTimeout` is better here

Reply:
change it to [`maxCheckPointTimeout`](https://github.com/pingcap/dm/pull/266/files#diff-013e2adce506b91024f15af06c7f7027R460)